deploy/crds/workspaces.ecd.eclipse.org_devworkspaces_crd.yaml

```diff
@@ -241,12 +237,15 @@ spec:
                                     path:
                                       type: string
                                     protocol:
-                                      default: tcp
+                                      description: The is the low-level protocol of
+                                        traffic coming through this endpoint. Default
+                                        value is "tcp"
```

Comment:
It looks like the default was removed? Similarly for `scheme`

Reply:
yes, because it is not supported by CRDs for now. It seems I had used a too-new kube-builder feature.